app/Http/Controllers/BlogController.php

```diff
@@ -113,7 +113,7 @@ public function update(Request $request, $slug)
             ]);
 
         return redirect(RouteServiceProvider::HOME)
-            ->with('message', 'Your post has been updated!');
+            ->with('message', 'Your post has been updated successfully!');
```

Comment:
Make the "successfully" "Successfully"

Reply:
you should have changed the pull title to something like "change success message in BlogController ....."
be direct cos that the essence of review 
what des "Make the "successfully" "Successfully"" means. even me I don't understand

Reply:
```suggestion
            ->with('message', 'Your post has been updated Successfully!');
```